pkg/query/storeset_test.go

```diff
@@ -521,7 +521,7 @@ func TestStoreSet_Update_NoneAvailable(t *testing.T) {
 	st.CloseOne(initialStoreAddr[0])
 	st.CloseOne(initialStoreAddr[1])
 
-	storeSet := NewStoreSet(nil, nil, func() (specs []StoreSpec) {
+	storeSet := NewStoreSet(nil, "default", nil, func() (specs []StoreSpec) {
```

Comment:
instead of passing `"default"` here, let's set it in the `NewStoreSet` constructor, and just pass empty string `""` here. 

Rationale: The fact that `"default"` is set is an implementation detail, constructor invokers can simply pass empty string and do not care, plus we won't forget to set `"default"` if we invoke `NewStoreSet` somewhere else.